ddtrace.gemspec

```diff
@@ -75,4 +75,15 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'datadog-ci', '~> 0.8.1'
 
   spec.extensions = ['ext/datadog_profiling_native_extension/extconf.rb', 'ext/datadog_profiling_loader/extconf.rb']
+
+  spec.post_install_message = <<-MSG
+    Thank you for installing ddtrace.
+
+    As of version 2, the `ddtrace` gem has been renamed to `datadog` to better represent the full suite of Datadog products.
+    The 1.x series will now only receive maintenance updates for security and critical bug fixes.
+
+    To upgrade, please update your Gemfile to replace `ddtrace` with `datadog`.
+
+    For detailed instructions on migration, see: https://github.com/DataDog/dd-trace-rb/blob/master/docs/UpgradeGuide2.md
```

Comment:
We should link this to a stable URL (and not `master` branch), because this message will be out in the wild for a long time.

I'm not sure what's the best git reference to pin this to, even the upgrade guide likely evolves overtime.

Or maybe he doesn't change too much that we can just link the specific comment? I'm not super sure.